src/components/listBox/ListBox.tsx

```diff
@@ -83,9 +84,14 @@ export class ListBox extends React.Component<IListBoxProps, {}> {
             />)
             .value();
 
-        return items.length
-            ? items
-            : <ItemBox {...{...this.props.noResultItem, classes: ['multi-line']}} />;
+        const emptyItem = (

```

Comment:
We can allowed a ReactNode as props for the empty box I think, Its more flexible if we want a specific design not related to a ItemBox

Reply:
I like the idea, not sure if it is a breaking change though, I will try it.

Reply:
It involves too much changes and its probably breaking, so I think its out of the scope of this fix. Would be nice to do though.